cmd/boulder-sa/main.go

```diff
@@ -25,6 +25,8 @@ type config struct {
 		Features map[string]bool
 	}
 
+	Features map[string]bool
```

Comment:
You should use  the `Features` field inside the `SA` struct instead (which was just added in #2199).